### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by jamrce**
## Description
Added some repo docs, including readme, to setup for next release.

## Key Changes 
  - Added readme
  - Added code of conduct
  - Added repo templates for: pull request, bug report and feature requests.

### CODE REVIEW

File: CODE_OF_CONDUCT.md
Comment (Kallb123) [inline CODE_OF_CONDUCT.md:117]: The links here seem malformed?

File: README.md
Comment (Kallb123) [inline README.md:3]: have*

File: README.md
Comment (Kallb123) [inline README.md:19]: Need to make a CONTRIBUTING.MD

**Review by Kallb123 — CHANGES_REQUESTED** [review]
Couple of tweaks required.

File: README.md
Comment (jamrce) [reply README.md:19]: Good spot.

File: CODE_OF_CONDUCT.md
Comment (jamrce) [reply CODE_OF_CONDUCT.md:117]: They were at the bottom using markdown linking but I'll put them inline if its causing an issue for you.

File: README.md
Comment (jamrce) [reply README.md:19]: Added one

**Review by twalmsley — APPROVED** [review]

File: README.md
Comment (jamrce) [reply README.md:5]: Thanks, I've tweaked this in my next commit.

File: README.md
Comment (jamrce) [reply README.md]: Good suggestion. Made changes for it to be forked. We can do our own branches as the core team but good practice for everyone else to fork.

File: README.md
Comment (jamrce) [reply README.md]: Solved.

File: CONTRIBUTING.md
Comment (amrc-benmorrow) [inline CONTRIBUTING.md:25]: If we're telling people to fork the repo I'm not sure we need to tell them how to name their branches?

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (amrc-benmorrow) [inline .github/ISSUE_TEMPLATE/bug_report.yml:22]: Since this is a library rather than an application, it might be better to request a minimal code example demonstrating the problem. Requesting explicitly 'what output did you get?' and 'what output did you expect?' is probably also a good idea.

File: CONTRIBUTING.md
Comment (amrc-benmorrow) [inline CONTRIBUTING.md:9]: Tyop (repoository).

File: README.md
Comment (amrc-benmorrow) [inline README.md:29]: Link to `license.md` vs `LICENSE` in the repo. We are English so it should be 'licence' in any case (and in the title above); I would suggest `LICENCE.md` unless there's a good reason not to.

**Review by amrc-benmorrow — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml
Comment (jamrce) [reply .github/ISSUE_TEMPLATE/bug_report.yml:22]: Good suggestion. I've tweaked a new version but don't believe there is a way of testing until its merged in?

File: CONTRIBUTING.md
Comment (jamrce) [reply CONTRIBUTING.md:9]: Good spot.

File: README.md
Comment (jamrce) [reply README.md:29]: Solved.

File: CONTRIBUTING.md
Comment (amrc-benmorrow) [reply CONTRIBUTING.md]: I don't think so. That can be discussed more privately with anyone who gets given write access to the main repo.

Comment (jamrce) [reply CONTRIBUTING.md]: Sounds sensible. Updated on next commit.

**Comment by jamrce** [comment]
Just updated the license as well to Apache 2.0 as discussed today.

**Pull request merged**
